history-of-quakers/modernized/02-preface.adoc

```diff
@@ -342,14 +342,14 @@ for that it was originally written in Latin seems to me very probable.
 But however this be, it appears plainly, that the author would not publicly be known;
 for the title seemed designedly composed so that the readers
 should believe W. Ames to be the author of it,
-namely: The Light on the Candlestick,
-serving for Illustration of the principal matters in the book called,
-The Mysteries of the Kingdom of God, etc. against Galenus Abrahamson and his Assenters,
+namely:_ The Light on the Candlestick,_
+serving for illustration of the principal matters in the book called,
+_The Mysteries of the Kingdom of God, Etc._ against Galenus Abrahamson and his Assenters,
```

Comment:
This is a little confusing, but I think this whole thing, starging with "The Light on..." and ending with ".. W. Ames" comprises a _single_ book title, and should be wrapped in `[.book-title]` instead of italics.